api/envoy/extensions/filters/http/stateful_session/v3/stateful_session.proto

```diff
@@ -0,0 +1,56 @@
+syntax = "proto3";
+
+package envoy.extensions.filters.http.stateful_session.v3;
+
+import "envoy/config/core/v3/extension.proto";
+import "envoy/config/core/v3/health_check.proto";
+
+import "udpa/annotations/status.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.filters.http.stateful_session.v3";
+option java_outer_classname = "StatefulSessionProto";
+option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/filters/http/stateful_session/v3;stateful_sessionv3";
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Stateful Session Filter]
+// This filter sets an override host based on the extensible session state and updates the
+// session state based on the final selected upstream host.
+// [#extension: envoy.filters.http.stateful_session]
+
+// [#comment:TODO(wbpcode): To support global feature enabled or disabled flag.]
```

Comment:
?

Reply:
> Should there be a global disabled flag or a runtime key reference in StatefulSession? My thinking is that for some upstreams stateful sessions might be required to function at all, but for other cases it's a performance improvement and operators might wish to roll it out gradually.

From this comment.

Reply:
I think if the code is only enabled when configured, you probably don't need a runtime flag unless there is some major change that is not config guarded. In any case, I don't think this note needs to be kept on the API itself.

Reply:
🤔 get it.